OasisDEX/oasis-borrow
# Notification component

# [Notification component](https://app.shortcut.com/oazo-apps/story/5094/notification-center-ui-component-create-notification-component)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added notification component
  
## How to test 🧪
  <Please explain how to test your changes>

- use feature toggle notification, click on notification widget and verify whether notification component handle all possible states (read, unread) and types (critical / non-critical)
